stripe/src/main/java/com/stripe/android/model/Source.java

```diff
@@ -298,6 +322,11 @@ public void setSourceTypeData(Map<String, Object> sourceTypeData) {
         mSourceTypeData = sourceTypeData;
     }
 
+    public void setTypeRaw(@NonNull @Size(min = 1) String typeRaw) {
+        mTypeRaw = typeRaw;
+        setType(UNKNOWN);
```

Comment:
if `typeRaw` is a supported type, should we set the type appropriately?

e.g. `setTypeRaw("bancontact") -> type = BANCONTACT`

I think if we implement things this way, you won't need the `toUsableType` function (instead, you could just use `mTypeRaw`).